spec/commands/admin/destroy_scoped_style_spec.rb

```diff
@@ -50,7 +50,7 @@ module Admin
           expect { subject.call }.to broadcast(:ok)
 
           expect(AwesomeConfig.find_by(organization:, var: :scoped_styles).value.keys).not_to include("foo")
-          expect(AwesomeConfig.find_by(organization:, var: :scoped_style_foo)).to be_nil
+          expect(AwesomeConfig.find_by(organization:, var: :scoped_styles_foo)).to be_nil
```

Comment:
```suggestion
          expect(AwesomeConfig.find_by(organization:, var: :scoped_style_foo)).to be_nil
```

sub-vars are always singular

Reply:
Done